src/accelerate/accelerator.py

```diff
@@ -1231,6 +1231,8 @@ def prepare_model(self, model: torch.nn.Module, device_placement=None):
                 any kind of mixed precision
             device_placement (`bool`, *optional*):
                 Whether or not to place the model on the proper device. Will default to `self.device_placement`.
+            apply_mixed_precision (`bool`, *optional*, defaults to `True`):
+                Whether or not to apply automatic mixed precision to the model based on `self.mixed_precision`.
```

Comment:
I was thinking more of a `mixed_precision_only` flag, so we don't need a new method.